library/psa_crypto_its.h

```diff
@@ -106,7 +107,8 @@ psa_status_t psa_its_set(psa_storage_uid_t uid,
 psa_status_t psa_its_get(psa_storage_uid_t uid,
                          uint32_t data_offset,
                          uint32_t data_length,
-                         void *p_data);
+                         void *p_data,
+                         size_t* p_data_length );
```

Comment:
Please put the `*` next to the variable name.

Reply:
Done.